plugins/bitwarden/help.md

```diff
@@ -18,15 +18,9 @@ Bitwarden is an integrated open source password management solution for individu
 
 # Requirements
 
-The following information are required for using this plugin:
 * Bitwarden Client ID
 * Bitwarden Client Secret
 
-Read more:
-[Bitwarden API Key](https://bitwarden.com/help/public-api/#authentication).
-
-Access to the Bitwarden Public API is available to customers on the Enterprise or Teams organizations plans.
```

Comment:
Wondering if it should be reverted or not 😄

Reply:
I couldn't see how to add paragraph section to the requirements in the plugin.spec.yaml, so I moved this into the troubleshooting section now. Do you think it should be kept here?

Reply:
That's fine to have that in `troubleshooting` 😄